test/app/voiceController.js

```diff
@@ -20,19 +31,154 @@ describe('Voice reqs', function() {
       .send(twilioRecordingRequest)
       .expect(200)
       .end(function(err, res) {
+        if (err) {return done(err)}
+        return OutboundVoiceMessages.findOneByAttribute(
+          'RecordingSid', 
+          RecordingSid
+        ).then((ovm) => {
+          should.exist(ovm)
+          done()
+        }).catch(done)
+      });
+  });
+
+  it('should accept a new ovm voice recording (mocked)', function(done) {
+    mock.enable()
+    let params = "?userId=2&clientId=1"
+    params += `&deliveryDate=${new Date().getTime()}`
+    params += `&type=ovm`
+    twilioAgent.post('/webhook/voice/save-recording/' + params)
+      .send(twilioRecordingRequest)
+      .expect(200)
+      .end(function(err, res) {
+        if (err) {return done(err)}
         return OutboundVoiceMessages.findOneByAttribute(
           'RecordingSid', 
-          'REde2dd4be0e7a521f8296a7390a9ab21b'
+          RecordingSid
         ).then((ovm) => {
-          console.log(ovm)
           should.exist(ovm)
+          mock.disable()
           done()
         }).catch(done)
       });
   });
 
+  it('should update status of ovm when applicable', function(done) {
+    twilioAgent.post('/webhook/voice/status/')
+      .send(twilioStatusUpdate)
```

Comment:
what does ovm stand for? o** voice mail? 

is this intended to check to update the status attribute in msgs table?